crates/zk_evm/src/opcodes/execution/log.rs

```diff
@@ -108,6 +108,11 @@ impl<const N: usize, E: VmEncodingMode<N>> DecodedOpcode<N, E> {
                         BlobSha256Format::normalize_for_decommitment(&buffer);
                     decommit_header = header;
                     decommit_preimage_normalized = normalized_preimage;
+                } else if Eip7702DelegationFormat::is_valid(&buffer) {
```

Comment:
Not sure if changes in this file and `decommitter.rs` are needed; mostly copy-pasted them from EVM implementation.